sig-service-catalog/charter.md

```diff
@@ -0,0 +1,128 @@
+# SIG Service Catalog Charter
+Service Catalog is a Kubernetes extension project that implements the [Open Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI).
+It allows application developers the ability to provision and consume cloud services natively from within Kubernetes.
+
+This charter adheres to the conventions described in the [Kubernetes Charter README](https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md).
+
+## Scope
+
+### In scope
+
+This SIG’s main goals are:
+- Support, and adhere to, the Platform requirements of the OSBAPI specification.
```

Comment:
Perhaps make "OSBAPI specification" a link to the spec's master version?

Reply:
did you want to address this one?

Reply:
heh, I added the link on line 14, not 12. Fixing